fe/src/components/SearchFilters/FiltersDropdown.tsx

```diff
@@ -5,32 +5,16 @@ import {
 import { ChevronDownIcon } from '@chakra-ui/icons';
 import { Category } from '../../utils';
 import { HomeContext } from '../../pages/Home';
-
-const CATEGORIES: Array<Category> = [
-  {
-    id: 0,
-    name: 'Cameras',
-  },
-  {
-    id: 1,
-    name: 'Laptops',
-  },
-  {
-    id: 2,
-    name: 'Tripods',
-  },
-  {
-    id: 3,
-    name: 'Lenses',
-  },
-];
+import { useGetCategoriesQuery } from '../../generated/graphql';
 
 interface Props {
   activeFilters: Array<number> // Currently as array of category ids.
 }
 
 const FiltersDropdown: React.FC<Props> = ({ activeFilters }: Props): JSX.Element => {
   const { setFilters } = React.useContext(HomeContext);
+  const categoriesQuery = useGetCategoriesQuery();
+  const categories: Array<Category> = categoriesQuery.data ? categoriesQuery.data.categories : [];
```

Comment:
If you don't feel like handling errors and load state in this pr, then at least log the error to the console if it exists.